pyERGM/utils.py

```diff
@@ -3,8 +3,10 @@
 from typing import Collection
 import numpy as np
 import networkx as nx
+from PIL.GimpGradientFile import linear
```

Comment:
what's this? `linear` doesn't seem to be used + `PIL`/`pillow` not in requirements.txt